src/base/bittorrent/torrenthandle.cpp

```diff
@@ -377,7 +380,7 @@ bool TorrentHandle::isAutoManaged() const
 #if (LIBTORRENT_VERSION_NUM < 10200)
     return m_nativeStatus.auto_managed;
 #else
-    return (m_nativeStatus.flags & lt::torrent_flags::auto_managed);
+    return bool {m_nativeStatus.flags & lt::torrent_flags::auto_managed};
```

Comment:
why not use the usual static_cast?

Reply:
In such cases, they lead to the same result (https://en.cppreference.com/w/cpp/language/direct_initialization). It's just shorter.
But if you strongly insist, I will change it.

Reply:
since torrent_flags_t has operator bool() I think static_cast is more suitable (or familiar for me).

Reply:
In fact, `operator bool()` allows you to do the conversion in both (`bool {T}` and `static_cast<bool>(T)`) ways. Otherwise, you wouldn't be able to do it without an intermediate cast to the underlying type. So "familiar for me" is the only your reason. Of course, I can change that to make you more comfortable.

Reply:
just keep the current, it is shorter, we also might ditch static_cast in the whole code base and prefer this way.